group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTest.java

```diff
@@ -15673,6 +15676,349 @@ public void testReplayStreamsGroupTopologyTombstone() {
         assertThrows(GroupIdNotFoundException.class, () -> context.groupMetadataManager.streamsGroup("bar"));
     }
 
+    @Test
+    public void testConsumerGroupHeartbeatOnStreamsGroup() {
```

Comment:
The test are pretty repetitive - like the ones for the other group types. I considered saving some LOC (by using private helper methods, and parametrized tests), but kept them at way, since it's going to be easier to refactor if each test is self-contained.